pkg/providers/vsphere/vsphere.go

```diff
@@ -1077,6 +1080,19 @@ func buildTemplateMapCP(clusterSpec *cluster.Spec, datacenterSpec v1alpha1.VSphe
 		"eksaSystemNamespace":                  constants.EksaSystemNamespace,
 		"auditPolicy":                          common.GetAuditPolicy(),
 	}
+	if features.IsActive(features.TaintsSupport()) {
+		var taints []corev1.Taint
+		if len(clusterSpec.Spec.ControlPlaneConfiguration.Taints) > 0 {
+			for _, taint := range clusterSpec.Spec.ControlPlaneConfiguration.Taints {
+				if taint.Effect != "NoExecute" {
```

Comment:
same for vsphere, why removing NoExecute? thought we add toleration on cloud controller